fedocal/__init__.py

```diff
@@ -1030,7 +1038,9 @@ def view_meeting_page(meeting_id, full):
         tzone=tzone,
         title=meeting.meeting_name,
         editor=editor,
-        from_date=from_date)
+        from_date=from_date,
+        reminder_at_options=APP.config.get('ICAL_REMINDER_AT_OPTIONS') or []
```

Comment:
No need for the ``or`` here, just finish the get:
``
reminder_at_options=APP.config.get('ICAL_REMINDER_AT_OPTIONS', [])
``
The second parameter is the default to return if the key is not in the dict ;-)